database/index.js

```diff
@@ -37,9 +37,16 @@ let saveUpdate = (incoming) => {
     })
 }
 
-const deleteUpdates = () => {
+var deleteUpdates = () => {
   return Updates.deleteMany({});
 }
 
+var retrieveUpdates = (incId) => {
+  return Updates.find({"itemId": incId});
+}
+
+
+
 module.exports.saveUpdate = saveUpdate;
 module.exports.deleteUpdates = deleteUpdates;
+module.exports.retrieveUpdates = retrieveUpdates;
```

Comment:
If you want to make things a bit cleaner and easier for yourself you can take advantage of ES6 object syntax and name your module exports like this `module.exports = {saveUpdate, deleteUpdates, retrieveUpdates}`